src/js/utils/dom.js

```diff
@@ -73,16 +72,19 @@ export function stopMonitoringUrlChanges() {
 }
 
 
-function onWindowFocus() {
-    store.dispatch(hasFocus(true));
-}
+let onWindowFocus;
+let onWindowBlur;
 
-function onWindowBlur() {
-    store.dispatch(hasFocus(false));
-}
+export function monitorBrowserState(dispatch) {
```

Comment:
any risk of this being called more than once? do we need some way to remove listeners?